CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format located [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+- Added Base code 1.0
+
 ### Breaking Changes
```

Comment:
technically you'd want to cleanup the changelog from skel to make it clear that the changelog starts anew with a new plugin. I can fix this up for you when I merge or you can fix it up, either way is fine by me.